modules/system/assets/js/snowboard/snowboard.extras.js

```diff
@@ -0,0 +1,13 @@
+import Flash from './extras/Flash';
+import FlashListener from './extras/FlashListener';
+import Transition from './extras/Transition';
+import AttachLoading from './extras/AttachLoading';
+import StripeLoader from './extras/StripeLoader';
+
+((snowboard) => {
+    snowboard.addModule('transition', Transition)
+    snowboard.addModule('flash', Flash);
+    snowboard.addModule('flashListener', FlashListener);
+    snowboard.addModule('attachLoading', AttachLoading);
+    snowboard.addModule('stripeLoader', StripeLoader);
+})(window.snowboard);
```

Comment:
```suggestion
((Snowboard) => {
    ```suggestion
    Snowboard.addPlugin('transition', Transition)
    Snowboard.addPlugin('flash', Flash);
    Snowboard.addPlugin('flashListener', FlashListener);
    Snowboard.addPlugin('attachLoading', AttachLoading);
    Snowboard.addPlugin('stripeLoader', StripeLoader);
})(window.Snowboard);
```